libs/application/core/src/lib/messages.ts

```diff
@@ -11,6 +11,21 @@ export const coreMessages = defineMessages({
     defaultMessage: 'Til baka',
     description: 'Back',
   },
+  buttonCancel: {
```

Comment:
Should these translations strings be in applications core? 
I would recommend using service portal core messages instead if those strings are only used there.

Reply:
I assumed that the cancel button translation was common. In fact it is more common than many translations in the core messages. So is this maybe old convention to put in the coreMessages instead messages on project bases?

Reply:
Yeah I agree it is common but the application system is not right for that either. All translations we have in service portal we keep inside service portal. That way we can control and translate more easily each module.

Reply:
Sure I will move them. They have been moved.